src/lib/strategy-card/StrategyCardBorder.js

```diff
@@ -0,0 +1,31 @@
+const { Border } = require("../../wrapper/api");
+
+class StrategyCardBorder extends Border {
```

Comment:
Can you add some jsdoc describing what this is?  By extending border I see it's a UI widget, but a bit more about what is does would be nice (does it include the inner UI too, etc).

Reply:
wish granted and also changed the class a lot to make the consumption easier.